packages/Meshfree/test/tstMovingLeastSquaresOperatorSimpleProblem.cpp

```diff
@@ -113,38 +117,12 @@ void checkResults( std::vector<double> const &values,
 TEUCHOS_UNIT_TEST_TEMPLATE_1_DECL( MovingLeastSquaresOperatorSimpleProblem,
                                    corner_cases, DeviceType )
 {
-    // No target point and no source point.
-    {
-        std::vector<std::array<double, DIM>> source_points = {};
```

Comment:
Why did you delete this?

Reply:
For two of the cases the code does not throw and the last one it throws inside Kokkos instead of DTK. Since the code still throws (from Kokkos) when you try call `apply`, I didn't really try to understand why it stopped throwing during the construction.